examples/geojson-wfs.js

```diff
@@ -1,4 +1,12 @@
 import 'ol/ol.css';
 import {apply} from 'ol-mapbox-style';
 
-apply('map', 'data/geojson-wfs.json');
+apply('map', 'data/geojson-wfs.json', {
+  transformRequest: (urlStr, type) => {
+    if (type === 'GeoJSON') {
+      const url = new URL(urlStr + '&transformRequest=true');
+      const req = new Request(url);
+      return req;
+    }
+  },
+});
```

Comment:
```suggestion
apply('map', 'data/geojson-wfs.json');
```

Reply:
This change does not make sense in the example, and you have covered it in the test already. So please revert this example change.